app/models/concerns/public_body_derived_fields.rb

```diff
@@ -41,7 +41,9 @@ def set_first_letter
 
   def update_url_name
     if changed.include?('name') || changed.include?('short_name')
```

Comment:
Check that these `changed` attributes still work correctly now that they don't exist on the `PublicBody`